assimilation_code/programs/obs_seq_to_netcdf/obs_seq_to_netcdf.f90

```diff
@@ -444,8 +448,14 @@ program obs_seq_to_netcdf
 
       if ( file_exist(ncName) .and. append_to_netcdf ) then
          ncunit = NC_Compatibility_Check(ncName, iepoch)
-      else
-         ncunit = InitNetCDF(ncName, iepoch)
+      else 
+         if (.not. epoch_file_created(iepoch)) then
+            write(*,*) 'creating file ', ncName
```

Comment:
the logic looks great.  for this output line i'd write it into a string (string1 or something) and then use the error_handler with E_MSG to output it.  looking through the rest of the source here, only "if (verbose)" messages use print or write to *.    other messages use the error_handler which writes both to stdout and to the log file.